### PR TITLE
Users/kavipriya/blamechanges

### PR DESCRIPTION
**Pull request by kaadhina**

### CODE REVIEW

File: RFCs/0024-Blame-Collector-Options.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
+# 0024 Blame collector options
+
+## Summary
+This note outlines the proposed changes for adding more options to blame data collector.
```
Comment (PBoraMSFT) [inline RFCs/0024-Blame-Collector-Options.md]: test blame data collector.

File: RFCs/0024-Blame-Collector-Options.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+
+## Motivation
+To enable blame as a first class option in vstest task, we need some additional options to be added to the existing blame collector
```
Comment (PBoraMSFT) [inline RFCs/0024-Blame-Collector-Options.md]: first-class option to support troubleshooting scenarios, including unattended execution in the pipeline...

File: RFCs/0024-Blame-Collector-Options.md [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+
+## Proposed Changes
+Here are the proposed changes for the blame data collector.
+
+Blame collector has an optional parameter called "CollectDump" which will collect a mini dump of the test host on an unexpected exit of the process.
+Optionally one may choose to specify key value pairs to override this:
+
+CollectAlways: If you choose to collect dump even when there is no process crash. It takes values true/false.
+DumpType: If you choose to collect a full process dump. It takes values full/mini.
```
Comment (PBoraMSFT) [inline RFCs/0024-Blame-Collector-Options.md]: mini/full.. mention which is default..

File: RFCs/0024-Blame-Collector-Options.md [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+
+### Help text
+
+Here is how the new command line help text will look like:
+
+--Blame|/Blame:[CollectDump];[Key1]=[Value1];[Key2]=[Value2]
```
Comment (PBoraMSFT) [inline RFCs/0024-Blame-Collector-Options.md]: mention actual option names instead of key

File: RFCs/0024-Blame-Collector-Options.md [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+      Runs the test in blame mode. This option is helpful in isolating the problematic test causing test host crash.
+      It creates an output file in the current directory as "Sequence.xml",
+      that captures the order of execution of test before the crash.
+      You may optionally choose to collect process dump for the test host.
```
Comment (PBoraMSFT) [inline RFCs/0024-Blame-Collector-Options.md]: clarify the connection between the process dump collection

File: RFCs/0024-Blame-Collector-Options.md [hunk 6]
```diff
@@ -0,0 +1,71 @@
+# 0024 Blame collector options
+
+## Summary
+This note outlines the proposed changes for adding more options to test blame data collector.
```
Comment (abhishkk) [inline RFCs/0024-Blame-Collector-Options.md]: `changes for adding more options to test blame data collector. `> `changes for enhancing blame data collector options.`

**Pull request merged**